Sources/EduIDExpansion/Flows/Home/HomeNavigationDelegate.swift

```diff
@@ -0,0 +1,9 @@
+import Foundation
+
+protocol HomeNavigationDelegate: AnyObject {
+    
+    func showPersonalInfoScreen()
+    func showSecurityScreen()
+    func showActivityScreen()
+    func showScanScreen()
```

Comment:
Always have a sender in a delegate method.

Reply:
I did this